src/main/java/org/sunbird/assessment/service/AssessmentServiceV2Impl.java

```diff
@@ -219,11 +221,32 @@ private void readAssessmentLevelData(Map<String, Object> assessmentAllDetail, SB
 
     private void readSectionLevelParams(Map<String, Object> assessmentAllDetail,
                                         Map<String, Object> assessmentFilteredDetail) {
+        List<Map<String, Object>> sectionResponse = new ArrayList<>();
+        List<String> sectionParams = cbExtServerProperties.getAssessmentSectionParams();
```

Comment:
make sure the section level param contains all required vale in application.properties